communication/codec_bytes_test.go

```diff
@@ -0,0 +1,77 @@
+package communication
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestCodecBytesInterface(t *testing.T) {
+	var _ Codec = NewCodecBytes()
```

Comment:
This test does nothing?

Reply:
This test check if struct implements wanted interface